wrapanapi/systems/msazure.py

```diff
@@ -766,6 +775,40 @@ def list_router(self):
     def disconnect(self):
         pass
 
+    def _list_resources(self, resource_group=None, hours_old=0):
+        """
+        Lists group resources older than hours_old
+        """
+
+        resources = []
+        now_time = datetime.utcnow().replace(tzinfo=pytz.utc)
+
+        for res in self.list_all_resources_by_resource_group(resource_group):
+            start_time = parser.parse(res.additional_properties['createdTime'])
```

Comment:
All of this can and should be skipped if `hours_old == 0`, otherwise we waste a bunch of time/resources when every resource will end up in the list.